include/tables/actor_table.h

```diff
@@ -0,0 +1,703 @@
+/**
+ * Actor Table
+ *
+ * DEFINE_ACTOR should be used for normal actors
+ *    - Argument 1: Name of the actor overlay segment in the spec (without the ovl_ part)
+ *    - Argument 2: Enum value for this actor
+ *    - Argument 3: Allocation type (normal, permanent or absolute)
+ *    - Argument 4: Actor name, as a string (used for debug prints)
```

Comment:
```suggestion
 * DEFINE_ACTOR should be used for normal actors
 *    - Argument 0: Name of the actor overlay segment in the spec (without the ovl_ part)
 *    - Argument 1: Enum value for this actor
 *    - Argument 2: Allocation type (normal, permanent or absolute)
 *    - Argument 3: Actor name, as a string (used for debug prints)
```
Any reason why tables are 1-index based? In my sfx table PR in MM, I've introduced the first table as 0-index, and presumably this will be cleaned up all at once in OoT. Unless there's a specific reason to document it as such?

Reply:
The only reason this is 1-indexed is because this part is a copy-paste from OoT :sweat_smile: